apps/mark/frontend/src/components/ms_either_neither_contest.tsx

```diff
@@ -327,11 +201,7 @@ export function MsEitherNeitherContest({
             gridArea: 'pick-one-label',
           }}
         >
-          <Prose>
-            <P weight="bold">
```

Comment:
Noticed these were adding unnecessary spacing to the UI, eating up precious real estate in `XL` mode.